packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -146,8 +146,12 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
         }
 
         let color: string | undefined;
-        if (note && value.startsWith(TAGS_HIERARCHY)) {
-          const [maybeColor, colorType] = NoteUtils.color({ note });
+        if (mode !== ProcMode.IMPORT && value.startsWith(TAGS_HIERARCHY)) {
+          const [maybeColor, colorType] = NoteUtils.color({
```

Comment:
see [[prefer returning objects instead of tuples|dendron.dev.style#prefer-returning-objects-instead-of-tuples]]